solr/views.py

```diff
@@ -4,6 +4,13 @@
 import json
 from models import Limits
 from sqlalchemy import or_
+from werkzeug import MultiDict
+try:
+    from cStringIO import StringIO
+except:
+    from StringIO import StringIO
```

Comment:
Why this try/except? Will cStringIO sometimes not be available?

Reply:
in older python insllations; i guess this is just an old habit